pkg/storage/driver/secrets.go

```diff
@@ -80,6 +81,12 @@ func (secrets *Secrets) Get(key string) (*rspb.Release, error) {
 // secret fails to retrieve the releases.
 func (secrets *Secrets) List(filter func(*rspb.Release) bool) ([]*rspb.Release, error) {
 	lsel := kblabels.Set{"owner": "helm"}.AsSelector()
+	if os.Getenv("helm-serviceName") != "" {
```

Comment:
The `pkg` directory in the Helm code is something other applications can import as a library and use in their own code. The `cmd` top level directory is for the Helm client application. Information passed to the client isn't read directly by the package. It needs to be passed in through Go. Environment variables are only used for system level information like knowing the Kuberentes context.

Also, environment variables must be in all uppercase letters and use _ as a separator.